__tests__/app.test.js

```diff
@@ -175,3 +175,62 @@ describe("Post comment", () => {
       });
   });
 });
+
+describe.only("Patch an article", () => {
```

Comment:
Should remove .only's before pushing, we want to make sure all of your tests are passing!